clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -1817,8 +1817,13 @@ static void AddTypeSpecifierResults(const LangOptions &LangOpts,
     Results.AddResult(
         Result("bool", CCP_Type + (LangOpts.ObjC ? CCD_bool_in_ObjC : 0)));
     Results.AddResult(Result("class", CCP_Type));
+    Results.AddResult(Result("_Coroutine", CCP_Type));
     Results.AddResult(Result("wchar_t", CCP_Type));
 
+    Builder.AddTypedTextChunk("_Coroutine");
+    Builder.AddChunk(CodeCompletionString::CK_HorizontalSpace);
+    Builder.AddInformativeChunk("_Coroutine test");
```

Comment:
what does this do

Reply:
This gives us the information when you hover over the keyword